avocado/core/jobdata.py

```diff
@@ -12,127 +12,152 @@
 # Copyright: Red Hat Inc. 2016
 # Author: Amador Pahim <apahim@redhat.com>
 
+"""
```

Comment:
This patch also changes the directory to `jobdata` instead of `replay`, which is user visible and quite important change. Would you please extract this change as well?